### DISCUSSION TITLE
Provide access to restricted corpora to users from other universities

### DISCUSSION THREAD

**Discussion by BeritJanssen** [UUDigitalHumanitieslab/I-analyzer#1581] [Ideas]
If we provide access through the SurfConnect Identity Provider, rather than UU's own, we could grant users from other universities access. This issue hangs on a lot of ifs, but *if* the university library from another university has also access to a given corpus incorporated in I-Analyzer (easiest case: has the same agreement with Gale), we could also give students and researchers from another university access to this corpus.

**Comment by lukavdplas** [comment]
For clarification:

- You mean SURFconext, right?
- I think I already asked this: would this only work for Dutch universities? Otherwise, this could be a good people & parliament feature.

**Comment by lukavdplas** [comment]
@BeritJanssen  is this still something that would have to be implemented specifically on I-analyzer? Or would it be implemented on the side of the identity provider?
